pressure ulcer

```diff
@@ -0,0 +1,77 @@
+-- pressure ulcer during FIRST icu stay 
+ 
+DROP MATERIALIZED VIEW IF EXISTS pressure_ulcer_first_icu ;
+CREATE MATERIALIZED VIEW  pressure_ulcer_first_icu AS
+
+with iepre as 
+(
+select iea.subject_id , iea.hadm_id , iea.icustay_id
+ ,iea.intime
+ ,iea.outtime
+  ,  ROW_NUMBER() over (PARTITION BY iea.subject_id ORDER BY iea.intime) as rn
+from mimiciii.icustays iea 
```

Comment:
Our convention is to not include the `mimiciii` schema name in queries

Reply:
removed